app/main.py

```diff
@@ -1,3 +1,78 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self: "Distance", km: int) -> None:
+        self.km = km
+
+    def __str__(self: "Distance") -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self: "Distance") -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self: "Distance", other: "Distance") -> "Distance":
```

Comment:
other can be int and float. Fix everywhere
```suggestion
    def __add__(self: "Distance", other: "Distance" | int | float) -> "Distance":
```

Reply:
Fixed